_data/projects/reviews.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: Reviews
+desc: >-
+  Reviews is a terminal UI Dashboard for monitoring requests for code review across several Github repositories and pull requests.
+site: https://github.com/apoclyps/reviews
+tags:
+- python
+- development
+- code
+- review
+- tui
+upforgrabs:
+  name: help wanted
+  link: https://github.com/apoclyps/reviews/labels/help%20wanted
+stats:
+  issue-count: 2
+  last-updated: '2021-09-21T22:12:00Z'
```

Comment:
@apoclyps These lines are added by our workflows to update all the projects with their `issue count` and `last updated`. You don't need to add it with your PR. Please remove it.